geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/messenger/OldMembershipInformation.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.distributed.internal.membership.gms.messenger;
+
+import java.util.Set;
+
+import org.jgroups.JChannel;
+
+import org.apache.geode.distributed.DistributedMember;
+
+class OldMembershipInformation {
```

Comment:
I wonder if this class should be named "MembershipInformation". It is returned from "getMembershipInfo". It also gives you both the old membership ids AND the channel. I'm not sure if it is always old but I don't understand the context of this code very well.

Reply:
I'll make that change.  The use of Object was isolating other packages from needing to know anything about JGroups.  That's no longer the case, so we can add stronger typing.